Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/SmoothScrollIntoView.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+    /// <summary>
+    /// Smooth scroll the list to bring specified item into view
+    /// </summary>
+    public static class SmoothScrollIntoView
```

Comment:
We should have this be a partial class with our ListViewExtensions and rename that file to be ListViewExtensions.AlternateRows (we could split out the stretch container if we wanted to a third file for organization too if we wanted).